src/buildercore/bootstrap.py

```diff
@@ -172,16 +172,25 @@ def update_sqs_stack(stackname, **kwargs):
     context = context_handler.load_context(stackname)
     setup_sqs(stackname, context.get('sqs', {}), pdata['aws']['region'])
 
-def unsub_sqs(stackname, context_sqs, region, dry_run=False):
+def unsub_sqs(stackname, cfn_sqs, region, dry_run=False):
```

Comment:
The second parameter is actually unused in the method body so this may not be apparent, but a `context` sub-dictionary is passed in, not anything that should be tied to CloudFormation. It comes from
```
 setup_sqs(stackname, context.get('sqs', {}), pdata['aws']['region'])
```